components/pedigree/resources/src/main/resources/pedigree/gene.js

```diff
@@ -62,7 +62,7 @@ define([
             var oldSymbol = this._symbol;
             var needUpdate = false;
             try {
-                var parsed = JSON.parse(response);
+                var parsed = JSON.parse(response.responseText);
```

Comment:
addressed in separate PR https://github.com/phenotips/phenotips/pull/2451/files

Reply:
PR with fix above was merged into master, this PR needs rebase

Reply:
or just remove this commit

Reply:
will be removed before merging in, just the bug may affect testing right now